scripts/install.sh

```diff
@@ -6,14 +6,17 @@ sudo bash scripts/java_install.sh
 yes | sudo apt-get -y --force-yes install curl
 yes | apt-get -y --force-yes install git
 
+
 #install maven
-#curl -O http://www-us.apache.org/dist/maven/maven-3/3.5.2/binaries/apache-maven-3.5.2-bin.tar.gz
-#sudo tar -xvzf apache-maven-3.5.2-bin.tar.gz
-#sudo mv apache-maven-3.5.2 /opt/maven 
-#export M2_HOME=/opt/maven
-#export PATH=${M2_HOME}/bin:${PATH}
+curl -O http://www-us.apache.org/dist/maven/maven-3/3.5.2/binaries/apache-maven-3.5.2-bin.tar.gz
+sudo tar -xvzf apache-maven-3.5.2-bin.tar.gz
+sudo mv apache-maven-3.5.2 /opt/maven 
+export M2_HOME=/opt/maven
```

Comment:
@raghavagrawal These exports are going to be effective only during the vagrant box installation. They will not be available for vagrant user after vagrant ssh login.

You need to put these exports of environment variables in `/home/vagrant/.bashrc` file. An example would be as follows.

echo 'export M2=$M2_HOME/bin' >> /home/vagrant/.bashrc